python/lsst/obs/metadata/observationInfo.py

```diff
@@ -151,10 +115,49 @@ def strippedHeader(self):
             del hdr[c]
         return hdr
 
+    def __eq__(self, other):
+        """Compares equal if standard properties are equal
+        """
+        if type(self) != type(other):
+            return False
+
+        for p in self._PROPERTIES:
+            # Use string comparison since SkyCoord.__eq__ seems unreliable
+            # otherwise.
```

Comment:
It might be. Not that I can fix it in time for us. I'm having a similar problem with Quantity `assertAlmostEqual` since there is no `__round__` implemented so I have to convert to a value before calling `assertAlmostEqual`.